docs/constitution/pyth-dao-constitution.md

```diff
@@ -1,60 +1,61 @@
 # [DRAFT] Pyth DAO Constitution Proposal
 
-# Terminology
+## Terminology
 
 - **Votable Token:** staked token in the governance contract
 - **Pyth Improvement Proposal (PIP)**
 - **Pyth DAO Treasury:** all tokens held in the governance smart contract that is directly governed by the Pyth DAO via on-chain voting
 - **Pyth Forum:** the system adopted by the DAO to manage PIP, organize the voting process and provide connectivity to the governance contract on the Solana Blockchain
 - **Pythian Multisig Wallet:** smart contract wallet signed by the elected members of the Pythian Council
 - **Price Feed Multisig Wallet:** smart contract wallet signed by the elected members of the Price Feed Council
+- **Operations Wallet:** smart contract wallet used by council members to submit PIPs on-chain
```

Comment:
it's not a smart contract wallet
Maybe here worth pointing out that the PIP is submitted to the council itself